api/src/main/java/org/openmrs/api/impl/ProgramWorkflowServiceImpl.java

```diff
@@ -10,28 +10,34 @@
 package org.openmrs.api.impl;
 
 import java.util.ArrayList;
+import java.util.Set;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Date;
 import java.util.List;
-import java.util.Set;
-
+import java.util.Map;
 import org.openmrs.Cohort;
 import org.openmrs.Concept;
 import org.openmrs.ConceptAnswer;
 import org.openmrs.ConceptStateConversion;
+import org.openmrs.Encounter;
 import org.openmrs.Patient;
 import org.openmrs.PatientProgram;
+import org.openmrs.PatientProgramAttribute;
 import org.openmrs.PatientState;
 import org.openmrs.Program;
+import org.openmrs.ProgramAttributeType;
 import org.openmrs.ProgramWorkflow;
 import org.openmrs.ProgramWorkflowState;
+import org.openmrs.validator.ProgramServiceValidator;
 import org.openmrs.api.APIException;
 import org.openmrs.api.ProgramNameDuplicatedException;
 import org.openmrs.api.ProgramWorkflowService;
 import org.openmrs.api.context.Context;
 import org.openmrs.api.db.ProgramWorkflowDAO;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
Where is this Autowired used?

Reply:
I removed the method that was using it, let me remove the import too.